include/sharg/detail/version_check.hpp

```diff
@@ -190,7 +190,7 @@ class version_checker
                               " " +
                               out_file.string() +
                               " " +
-                              std::string{"http://seqan-update.informatik.uni-tuebingen.de/check/SeqAn3_"} +
+                              std::string{"http://seqan-update.informatik.uni-tuebingen.de/check/SeqAn-Sharg_"} +
```

Comment:
```suggestion
                              std::string{"https://seqan-update.informatik.uni-tuebingen.de/check/SeqAn-Sharg_"} +
```

Can you check whether `https` works?
The server seems to support it now.